snappi_ixnetwork/trafficitem.py

```diff
@@ -1129,6 +1130,22 @@ def _configure_size(self, ce_dict, size):
                 ce["frameSize"]["incrementFrom"] = size.increment.start
                 ce["frameSize"]["incrementTo"] = size.increment.end
                 ce["frameSize"]["incrementStep"] = size.increment.step
+            elif size.choice == "weight_pairs":
+                if size.weight_pairs.choice == "predefined":
```

Comment:
are we add test for it. Please add and validate it properly. without test it is quite difficult to find the issue.

Reply:
imix tests were added and running in CI.